examples/audio_server/server.cc

```diff
@@ -0,0 +1,132 @@
+#include <memory>
+#include <mutex>
+
+#include "../../blocks/http/api.h"
+#include "../../bricks/dflags/dflags.h"
+#include "../../bricks/sync/waitable_atomic.h"
+
+DEFINE_uint16(port, 8080, "The local port to use.");
+DEFINE_uint32(n, 3, "Max number of audio streams");
+
+const std::string cmd_kill = "kill";
+
+struct SharedState final {
+  uint32_t active_channels = 0;
+  std::map<std::string, std::string> channel_control;
+  bool channel_exists(std::string channel_id) const{
```

Comment:
`const std::string&` or `std::string const&`, since just `std::string` as an argument makes a redundant copy.

Btw, maybe your code is better off in a separate repo, using the `CMakeLists.txt` from `C5T/Current/cmake/`, such as https://github.com/dkorolev/c5t_trivial_dep_example ?

Reply:
Yes, because it's just using current. Let's do the review and I can move it to the separate repo

Reply:
done